From eef2b7f5dd51a0b27cb291fccb995f1e3630b11f Mon Sep 17 00:00:00 2001
From: Laszlo Ersek <lersek@redhat.com>
Date: Thu, 21 Nov 2013 16:27:17 +0100
Subject: [PATCH 07/14] introduce RFQDN_REDHAT (RHEL 6->7 fwd)

RH-Author: Laszlo Ersek <lersek@redhat.com>
Message-id: <1385051239-3677-2-git-send-email-lersek@redhat.com>
Patchwork-id: 55836
O-Subject: [RHEL-7.0 qemu-kvm PATCH 1/3] introduce RFQDN_REDHAT (RHEL 6->7 fwd)
Bugzilla: 971933
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Jiri Denemark <jdenemar@redhat.com>
RH-Acked-by: Luiz Capitulino <lcapitulino@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=971933

This forward ports the final effect of RHEL-6 commit

  commit 8d99c4ad4d9fa53768bfbca8606457f8179f4404
  Author: Luiz Capitulino <lcapitulino@redhat.com>
  Date:   Fri May 14 22:49:19 2010 -0300

      Monitor: Make RFQDN_REDHAT public

Signed-off-by: Laszlo Ersek <lersek@redhat.com>
---
 include/monitor/monitor.h | 3 +++
 1 file changed, 3 insertions(+)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 include/monitor/monitor.h |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/include/monitor/monitor.h b/include/monitor/monitor.h
index 10fa0e3..dcce609 100644
--- a/include/monitor/monitor.h
+++ b/include/monitor/monitor.h
@@ -19,6 +19,9 @@ extern Monitor *default_mon;
 /* flags for monitor commands */
 #define MONITOR_CMD_ASYNC       0x0001
 
+/* Red Hat Monitor's prefix (reversed fully qualified domain) */
+#define RFQDN_REDHAT "__com.redhat_"
+
 /* QMP events */
 typedef enum MonitorEvent {
     QEVENT_SHUTDOWN,
-- 
1.7.1

